src/components/Footer.jsx

```diff
@@ -1,20 +1,175 @@
 import React from 'react'
+import { Link } from 'gatsby'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
 import styled from 'styled-components'
-import logo from '../images/cfc_logo_short.svg'
+import longArrowAltRight from '../images/join-us/long-arrow-alt-right.svg'
 
 const StyledFooter = styled.footer`
-  width: 14rem;
+  width: 100%;
+  padding: 1rem 0;
```

Comment:
Looks like the padding is 3rem in Figma. Let's update that. 
<img width="674" alt="Screen Shot 2023-01-15 at 7 21 33 PM" src="https://user-images.githubusercontent.com/6326891/212579520-70b33577-e112-42b7-a8de-ff60be35767d.png">